src/frontend/web_application/src/components/brightForm/CheckboxFieldGroup/index.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { v1 as uuidV1 } from 'uuid';
+import Checkbox from '../Checkbox';
+import Switch from '../../form/CheckboxFieldGroup/components/Switch';
+import { FieldErrors } from '../';
+
+import './style.scss';
+
+class CheckboxFieldGroup extends PureComponent {
```

Comment:
What do you think about changing the way we build input form, may be something like:

```
<FieldGroup errors={errors}><Checkbox ... /><Label ... /></FieldGroup>
```